src/Controllers/QuestionTemplateController.php

```diff
@@ -1,68 +1,118 @@
 <?php
 
-
 namespace QuizApp\Controllers;
 
-
 use Framework\Contracts\RendererInterface;
 use Framework\Controller\AbstractController;
 use Framework\Http\Request;
 use Framework\Http\Response;
 use Framework\Http\Stream;
+use QuizApp\Services\Paginator;
 use QuizApp\Services\QuestionTemplateServices;
 use QuizApp\Services\QuizTemplateServices;
+use Symfony\Component\DependencyInjection\Exception\ParameterNotFoundException;
 
+/**
+ * Class QuestionTemplateController
+ * @package QuizApp\Controllers
+ */
 class QuestionTemplateController extends AbstractController
 {
 
+    /**
+     * @var QuestionTemplateServices
+     */
     private $questionTemplateServices;
 
+    /**
+     * @var Paginator
+     */
+    private $paginator;
+
     /**
      * UserController constructor.
      * @param RendererInterface $renderer
      * @param QuestionTemplateServices $questionInstanceServices
+     * @param Paginator $paginator
      */
-    public function __construct(RendererInterface $renderer, QuestionTemplateServices $questionInstanceServices)
+    public function __construct(RendererInterface $renderer, QuestionTemplateServices $questionInstanceServices, Paginator $paginator)
     {
         parent::__construct($renderer);
         $this->questionTemplateServices = $questionInstanceServices;
+        $this->paginator = $paginator;
     }
 
+    /**
+     * This method creates a user and saves it in the database
+     *
+     * @param Request $request
+     * @return Response
+     */
     public function createQuestion(Request $request)
     {
         $text = $request->getParameter('text');
         $type = $request->getParameter('type');
         $answer = $request->getParameter('answer');
-        $this->questionTemplateServices->saveQuestion($text, $type,$answer);
+        $this->questionTemplateServices->saveQuestion($text, $type, $answer);
         $location = 'Location: http://quizApp.com/admin-questions-listing?page=1';
```

Comment:
Not sure I've mentioned this before, but try to find a way to avoid hardcoding page=1 (e.g. if a page is not given, your app should behave as if the page is 1).